src/modules/blocks/ticket/container-content/attendee-collection/style.pcss

```diff
@@ -0,0 +1,74 @@
+.tribe-editor__ticket__attendee-collection {
+	margin-top: 20px;
+
+	.tribe-editor__accordion__row {
+		position: relative;
+
+		&.active {
+
+			&:before {
+				content: '';
+				height: 1px;
+				width: calc(100% + 12px);
+				background-color: #E1E3E6;
+				position: absolute;
+				top: 10px;
+				left: 0;
+			}
+		}
+
+		&.open {
+			.tribe-editor__accordion__row-content {
+				overflow: visible;
+			}
+		}
+
+		.tribe-editor__accordion__row-content {
```

Comment:
You added the `tribe-editor__ticket__attendee-collection-content` in the accordion settings, better to use that than target the general accordion row content class.